### PR TITLE
Homepage alpin

### PR DESCRIPTION
**Pull request by alpinmohsa541**
**Changes**

- create header
- create homepage
- create a column search schedule
- create a calendar in the flight schedule search column

**Todo**

- Responsive
- Create a class column in the schedule search field

**Preview**
Tampilan  Dekstop
image
Tampilan Calender di bagian pencarian jadwal penerbangan
image
Tampilan jumlah penumpang dan jenis penumpang
image

### CODE REVIEW

File: src/pages/Homepage/Homepage.js [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
 
-export default function Homepage() {
-  return (
-    <div>Homepage</div>
+const Homepage = () => {
+  return(
+    <div>
+    <Navbar/>
+    <Header/>
+    </div>
```
Comment (saepasomba) [inline src/pages/Homepage/Homepage.js:12]: Indentasinya

File: src/index.js [hunk 2]
```diff
@@ -1,18 +1,27 @@
...
-    <BrowserRouter>
-      <Routes>
-        <Route path='/' element={<Homepage />} />
-      </Routes>
-    </BrowserRouter>
+    <ChakraProvider>
+      <BrowserRouter>
+        <Routes>
+          {/* <Route path='/' element={<Navbar />} /> */}
+          <Route path='/' element={<Header />} />
```
Comment (saepasomba) [inline src/index.js:21]: Mungkin ini ke Homepage?

File: src/components/navbar/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+export const Navbar = () => {
+  return (
+    <div className="navbar">
+      <div className="navContainer">
+        <span className="logo">BinarFly</span>
+        <div className="navItems">
+        <Button colorScheme='blue'>Register</Button>
+        <Button colorScheme='blue'>Login</Button>
+        </div>
```
Comment (saepasomba) [inline src/components/navbar/Navbar.js:13]: Indentasinya

**Review by saepasomba — CHANGES_REQUESTED** [review]
Berhubung diliat-liat masnya make pure HTML CSS, saya saranin pake CSS module [(link)](https://css-tricks.com/css-modules-part-1-need/) ** Bisa cek resource lain. Ini supaya class name nya gak tabrakan sama module / CSS framework lain.

**Comment by alpinmohsa541** [comment]
> Berhubung diliat-liat masnya make pure HTML CSS, saya saranin pake CSS module [(link)](https://css-tricks.com/css-modules-part-1-need/) ** Bisa cek resource lain. Ini supaya class name nya gak tabrakan sama module / CSS framework lain.

**Pull request closed without merging**